portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,60 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <style>
+body {
+  background-image: url('images/white-cat-1.png');
+}
+</style>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>The Portfolio</h1>
+      <b>(Scroll to the bottom for a surprise)</b>
+      <p>Hello, I am Anthony Achille, Check out my <a href="https://github.com/Anthony869?tab=repositories">GitHub</a></p>
+      <p></p>
+      <p></p>
+      <button onclick="addRandomGreeting()">Ricky Quotes</button>
       <div id="greeting-container"></div>
+      <p></p>
+      <p></p>
+      <strong>Background:</strong>  
+      <p>I have always been interested in programming and computers in general. Ever since I could remember I wnated to learn
+         more about computer and how they work. Technology has always been interesting and being able to control them with code.
+         As soon as I could take a class about computers in highschool I did, I took Computer Science 3 the first year and continued 
+         with the path after that. Before I could do that at school I went to camps over the summer to learn different languages each time.
+        </p>
+      <p>I also like to Golf and have for 9+ years. I have been playing since I was like 6 or 7 because my Nonno introduced me to it.
+         I've also played basketball and baseball but I didn't like them as much as I liked Golf. I tried out for the golf team the
+         first year I could when I was a freshman in highschool. The next year I did the Junior Varsity with a bunch of my friends.
+         I did the same thing this year with all my friends that golf for the very end of the season. 
```

Comment:
You might want to consider rewording this sentence, it is a little hard to totally grasp.

Reply:
Still not sure that "I did the same thing this year my golfing friends for a couple games. " is clear. Read it aloud, and let me know.